utils/dataset_manifest/README.md

```diff
@@ -0,0 +1,71 @@
+## Simple command line to prepare dataset manifest file
+
+### Steps before use
+
+When used separately from Computer Vision Annotation Tool(CVAT), the required modules must be installed
+
+```bash
+python3 -m venv .env
+. .env/bin/activate
+pip install -U pip
+pip install -r requirements.txt
```

Comment:
@Marishka17 , Is it enough only to install python dependencies? About about ffmpeg and other packages?

Can we have an alternative way to run the tool using openvino/cvat_server docker image on DockerHub?

Reply:
I think, that it is possible to use it with openvino/cvat_server, but initially, the task was to separate
this functionality as much as possible from CVAT. Ok, I will update the documentation.